tests/testthat/test-slice-chop.R

```diff
@@ -429,10 +417,24 @@ test_that("name repair is respected and happens after ordering according to `ind
   expect_named(vec_unchop(x, indices, name_repair = "unique"), c("a...1", "a...2"))
 })
 
-test_that("errors on OOB zero values", {
-  skip("until we can error on 0 values in vec_as_location()")
+test_that("errors on `0` locations", {
```

Comment:
Can you merge the two subscript type tests in one block and wrap in `verify_errors()` please?